app/importers/rows/course_row.rb

```diff
@@ -0,0 +1,29 @@
+class CourseRow < Struct.new(:row, :school_ids_dictionary)
+  # Represents a row in a CSV export from Somerville's Aspen X2 student information system.
+  # This structure represents courses ignoring the section data. Unfortunately this is run 
+  # for each section, rather than each course.
```

Comment:
@kevinrobinson and I had talked about this. The unfortunate bit is that it will re-query a course for each of its sections. No big deal though.